src/components/Dropdown/DropdownContent.tsx

```diff
@@ -28,13 +31,14 @@ export const DropdownContent: React.FC<Props> = ({
 
   return (
     <DropdownContentRoot>
-      <DropdownContentContext.Provider value={{ onClickCloser }}>
+      <DropdownContentContext.Provider value={{ onClickCloser, controllable, scrollable }}>
         <DropdownContentInner
           triggerRect={triggerRect}
           scrollable={scrollable}
           className={`${dropdownKey} ${className}`}
+          controllable={controllable}
         >
-          {controllable ? children : <DropdownCloser>{children}</DropdownCloser>}
```

Comment:
I had to pass a value from DropdownContentInner. So we move this logic to DropdownContentInner